### DISCUSSION TITLE
Velocity inquiry

### DISCUSSION THREAD

**Discussion by WilMicro** [FundamentalFrequency/.github#24] [Q&A]
What code is there for velocity detection (from two contact switches) being calculated & appended to the midi on message?

**Comment by stonepreston** [comment]
The LMN-3 does not currently have any such feature. The velocity of the midi notes is static. However, Control Surface, the library that provides the midi functionality, does [support setting velocity of individual note buttons](https://tttapa.github.io/Control-Surface-doc/Doxygen/de/d36/classBankable_1_1NoteButton.html#a1bd7c42ec152adde0cacef9486418c40) if you were wanting to go that route.

**Reply by WilMicro** [reply]
Good to know. Thanks
